### DISCUSSION TITLE
New user `decky` for decky?

### DISCUSSION THREAD

**Discussion by tony-sol** [SteamDeckHomebrew/decky-loader#579] [Q&A] [answered]
Is there a way to install decky along with new decky user, like homebrew for linux does?
I think it may be more clear install separate from main deck user

**Comment by TrainDoctor** [comment]
Decky Loader is Deck first, everything else second. I'm not sure what the point of this would be. The software already supports installing to home directory of any user.

**Reply by tony-sol** [reply]
It's more about convenience and consistency. 
If i understand it right, Deckly Loader is silimar to package manager by design.
Usually, package managers either has user-independent storage and inject all artefact to user namespace, or provides separate user.
Or as long it's arch-based distro, it may follow XDG Spec.

To exclude XY problem, i actually just want to keep my `$HOME` clean as much as possible - if there is a way to move `/home/deck/homebrew` to eg. `"${XDG_DATA_HOME}/decky"` and preserve config into `"${XDG_CONFIG_HOME}/decky"` or similar - i'd be really happy :)

**Reply by TrainDoctor** [reply]
If you would like to add an xdg compliant install option to the launcher and systemd service files we use then I would be happy to accept those contributions. Decky is not a package manager and very explicitly disallows plugins that are cross-dependent in our review process to ensure simplicity in review and simplicity in plugins that users get.

I understand your reasoning but given decky's use case and our explicit goal being feature functionality for the steam deck made easier rather than simply a Gamemode UI patcher, this is not a priority for us.

**Reply by tony-sol** [reply]
> Decky is not a package manager

Maybe it's me, but i see it really similar with eg. lazy.nvim or mason.nvim - not a full-scale system-wide powertool, like homebrew/apt/pacman, but still: it provides repository for store and interface for manage (download/install/update/uninstall) ~~packages~~ plugins, not for common OS usage, but for specific tool - steam.

Anyway, it's just thinking out loud and i'll be happy to contribute if i can :)

Thank you!

**Comment by AAGaming00** [comment, marked as answer]
Adding new users on SteamOS tmk requires disabling read-only mode so we can't support it there, and since we're deck-first it's just not worth the complexity.

**Reply by Elderon-TD** [reply]
would this also mean there would be no way/interest in adding a way to access/manage plugins while in desktop mode (how I use my Deck ~75% of the time)?
